solution/backend/cmcs_regulations/settings/base.py

```diff
@@ -50,7 +50,8 @@
     'django.contrib.sitemaps',
     'django.contrib.syndication.views',
     'django_jsonform',
-    'file_manager'
+    'file_manager',
+    'content_search',
```

Comment:
search is its own little app for resources.  This is still independent of regcore search